ci-operator/config/openshift-assisted/cluster-api-provider-openshift-assisted/openshift-assisted-cluster-api-provider-openshift-assisted-master.yaml

```diff
@@ -21,6 +21,12 @@ build_root:
       rm -rf /tmp/kubectl && \
       dnf install -y python3-pip diffutils && \
       pip install ansible ansible-lint jinja2-cli
+      mkdir -p "$HOME/.local/bin"
+      curl -L https://github.com/mike-engel/jwt-cli/releases/download/6.2.0/jwt-linux.tar.gz -o "jwt-linux.tar.gz"
+      tar -xzf "jwt-linux.tar.gz"
+      mv "jwt" "$HOME/.local/bin/"
+      chmod +x "$HOME/.local/bin/jwt"
+      ENV PATH="$PATH:$HOME/.local/bin"
```

Comment:
no need to create a new directory to export, just make sure we install it in a directory already in the PATH